notebook/static/notebook/js/cell.js

```diff
@@ -299,6 +299,7 @@ define([
                     this.element.removeClass('marked');
                 }
             }
+            this.events.trigger('marked_changed.Cell', {cell:this, value:value});
```

Comment:
let's move that 1 line up to be more efficient and not trigger event if we don't change the status.